draft-ietf-scitt-architecture.md

```diff
@@ -297,7 +297,8 @@ Transparency is implemented by providing a consistent, append-only, cryptographi
 A SCITT instance is referred to as a Transparency Service.
 Implementations of Transparency Services may protect their Append-only Log using a combination of trusted hardware, replication and consensus protocols, and cryptographic evidence.
 A Receipt is an offline, universally-verifiable proof that an entry is recorded in the Append-only Log.
-Receipts do not expire, but it is possible to append new entries (more recent Signed Statements) that subsume older entries (less recent Signed Statements).
+Receipts do not have an expiration time, but the corresponding key material could.
```

Comment:
This addition contradicts [draft-ietf-cose-merkle-tree-proofs](https://datatracker.ietf.org/doc/html/draft-ietf-cose-merkle-tree-proofs-05#name-validity-period), aka COSE Receipts, although that is an informative reference, it seems pretty important.

```suggestion
```

Reply:
The content of the Merkle tree proof is immutable and does not expire.  The ability to ask CA's and such later the same questions becomes problematic and why that information should be part of the information on the Ledger.  This is immutable and like your kindergarten signature it is still you.  This is what the above reference is about.  The operation of signing the proof with a new signature does not diminish that statement.

Reply:
> The content of the Merkle tree proof is immutable and does not expire.

That is not what the referenced I-D says, so I will copy-paste the section I cited here for clarity.

> In some cases, receipts MAY include strict validity periods, for example, activation not too far in the future, or expiration, not too far in the past. See the iat, nbf, and exp claims in [[RFC8392](https://www.rfc-editor.org/rfc/rfc8392)], for one way to accomplish this. The details of expressing validity periods are out of scope for this document.

The claims MAY include strict validity periods. That is not about the digital signature itself, but the claim properties signed. It not a MUST however.

If SCITT were to say that receipts MUST NOT have an expiration time (i.e. use those fields) and clarify how it takes the general capabilities of a COSE-wrapped CWT and strictly forbids the use of those, I would say that explanation is much clearer to implementers about what needs to be done and meet the requirement you want to add.

I can propose wording to that effect, does that make sense?

Reply:
I support applying this change